src/utils/rehype-code-mockup.ts

```
@@ -0,0 +1,115 @@
+import type { ShikiTransformer } from 'shiki';
+import type { Element, ElementContent } from 'hast';
+import { h } from 'hastscript';
+import { toString } from 'hast-util-to-string';
```

Biome (2.3.13)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)